dash/dash.py

```diff
@@ -677,6 +677,14 @@ def serve_component_suites(self, package_name, path_in_package_dist):
             'map': 'application/json'
         })[path_in_package_dist.split('.')[-1]]
 
+        loader = pkgutil.get_loader(package_name)
+        self.logger.info(
+            "serving package %s resource %s => %s",
+            package_name,
+            path_in_package_dist,
+            loader.path if hasattr(loader, 'path') else loader,
```

Comment:
python 2 has different `loader` protocol, so we will see the path only in python3

Reply:
[elsewhere](https://github.com/plotly/dash/blob/a42b712e1a88d17400726b5b28ef20a36e826d21/dash/development/base_component.py#L22) we just reference `sys.modules[package_name]` - can we just get the package there and grab its `__path__`?

Reply:
tried in python 2.7, I think it's basically the same, as the protocol is at the lower level, so you won't even get `__path__` from the loader object

Reply:
Ah I see, you're right, `module.__path__` and `module.__file__` *exist* but they aren't really paths.